tests/ffi/sysctl/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+# Download ffi-tools container and push ffi-tools image into QM registry
+prepare_images
+
+# Run the ffi-tools container in qm vm
+run_container_in_qm ffi-qm
+
+# Get numbers of sysctl permission denied
+sysctl_num=$(podman exec qm /bin/bash -c \
+               "podman run  --replace --name ffi-qm  dir:${QM_REGISTRY_DIR}/ffi-tools:latest \
+               ./setsysctl 2>&1" | grep -c "sysctl: permission denied on key")
```

Comment:
the container already run in 
`run_container_in_qm ffi-qm`
you just need to exec command on ffi-qm that already running detached container

replace this 
podman run --replace --name ffi-qm dir:${QM_REGISTRY_DIR}/ffi-tools:latest  -->

podman exec ffi-qm shell -c "./setsysctl 2>&1" | grep -c "sysctl: permission denied on key")

Reply:
Oh, yes, you are right. Thanks!